data/registers/octospi_v1.yaml

```diff
@@ -0,0 +1,699 @@
+    bit_offset: 4
+    bit_size: 2
+    enum: SizeInBits
+  - name: ADMODE
+    description: 'Address mode. This field defines the address phase mode of operation. 101-111: Reserved'
+    bit_offset: 8
+    bit_size: 3
+    enum: PhaseMode
+  - name: ADDTR
+    description: Address double transfer rate. This bit sets the DTR mode for the address phase.
+    bit_offset: 11
+    bit_size: 1
+  - name: ADSIZE
+    description: Address size. This field defines address size.
+    bit_offset: 12
+    bit_size: 2
+    enum: SizeInBits
+  - name: ABMODE
+    description: 'Alternate-byte mode. This field defines the alternate-byte phase mode of operation. 101-111: Reserved'
+    bit_offset: 16
+    bit_size: 3
+    enum: PhaseMode
+  - name: ABDTR
+    description: Alternate bytes double transfer rate. This bit sets the DTR mode for the alternate bytes phase. This field can be written only when BUSY = 0.
+    bit_offset: 19
+    bit_size: 1
+  - name: ABSIZE
+    description: Alternate bytes size. This bit defines alternate bytes size.
+    bit_offset: 20
+    bit_size: 2
+    enum: SizeInBits
+  - name: DMODE
+    description: 'Data mode. This field defines the data phase mode of operation. 101-111: Reserved'
+    bit_offset: 24
+    bit_size: 3
+    enum: PhaseMode
+  - name: DDTR
+    description: Data double transfer rate. This bit sets the DTR mode for the data phase.
+    bit_offset: 27
+    bit_size: 1
+  - name: DQSE
+    description: DQS enable. This bit enables the data strobe management.
+    bit_offset: 29
+    bit_size: 1
+  - name: SIOO
+    description: Send instruction only once mode. This bit has no effect when IMODE = 00 (see ).
+    bit_offset: 31
+    bit_size: 1
+fieldset/CR:
+  description: control register
+  fields:
+  - name: EN
+    description: 'Enable This bit enables the OCTOSPI. Note: The DMA request can be aborted without having received the ACK in case this EN bit is cleared during the operation. In case. this bit is set to 0 during a DMA transfer, the REQ signal to DMA returns to inactive state without waiting for the ACK signal from DMA to be active.'
+    bit_offset: 0
+    bit_size: 1
+  - name: ABORT
+    description: 'Abort request. This bit aborts the ongoing command sequence. It is automatically reset once the abort is completed. This bit stops the current transfer. Note: This bit is always read as 0.'
+    bit_offset: 1
+    bit_size: 1
+  - name: DMAEN
+    description: 'DMA enable In Indirect mode, the DMA can be used to input or output data via DR. DMA transfers are initiated when FTF is set. Note: Resetting the DMAEN bit while a DMA transfer is ongoing, breaks the handshake with the DMA. Do not write. this bit during DMA operation.'
+    bit_offset: 2
+    bit_size: 1
+  - name: TCEN
+    description: Timeout counter enable. This bit is valid only when the Memory-mapped mode (FMODE[1:0] = 11) is selected. This bit enables the timeout counter.
+    bit_offset: 3
+    bit_size: 1
+  - name: DQM
```

Comment:
this is an SVD mistake, the L4+ RM also calls this DMM, in-line with the other versions

Reply:
I thought I caught that, but it slipped through.